langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -190,7 +190,7 @@ function testLastIndexOf() {
 
 function testReverse() returns [int[], int[]] {
     int[] arr = [10, 20, 30, 40, 50];
-    return [arr, arr.reverse()];
+    return [arr.reverse()];
```

Comment:
Instead of passing the result to java and asserting it there, we can assert it in ballerina using the `ballerina/test` module